include/mxnet/lib_api.h

```diff
@@ -393,13 +458,21 @@ class OpResource {
     return static_cast<mx_stream_t>(cuda_stream);
   }
 
+  void alloc_ndarray(MXSparse* sparse, int index, int indices_len, int indptr_len = 0) {
+    ndarray_malloc(ndarray_alloc, index, indices_len, indptr_len, 
```

Comment:
maybe call this "sparse_malloc" so that for us developers its easier to follow?

Reply:
Fixed.